scripts/exgdas_global_marine_analysis_run.sh

```diff
@@ -108,6 +132,13 @@ concatenate_bump 'bump3d'
 clean_yaml var.yaml
 $APRUN_SOCAANAL $JEDI_BIN/soca_var.x var.yaml 2>var.err
 
+
+# increment update for MOM6 iau
+# Make sure the last (most recent) increment file put into socaincr2mom6 as $1
+( socaincr2mom6 `ls -t ${DATA}/Data/ocn.*iter*.incr* | head -1` ${DATA}/INPUT/MOM.res.nc ${DATA}/soca_gridspec.nc ${DATA}/Data/inc.nc )
```

Comment:
My bad for remembering now @hyunchul386, but we are also saving the full increment ([yaml snippet here](https://github.com/NOAA-EMC/GDASApp/blob/d7c822b0e910ba49efedab3993f71980334d8704/parm/soca/variational/3dvarfgat.yaml#L75)) . Can you point to that increment instead? 
It's in 
```./Data/ocn.3dvarfgat_pseudo.incr.2018-04-15T09:00:00Z.nc```